.openpublishing.redirection.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This looks right. I don't know if the relative paths are correct or not - does it build without warnings?